setup-network-registry.sh

```diff
@@ -24,7 +25,10 @@ rm -r ./config/*/nuts.yaml
 cp ./config/bundy/nuts.yaml.template ./config/bundy/nuts.yaml
 cp ./config/dahmer/nuts.yaml.template ./config/dahmer/nuts.yaml
 
-nohup docker-compose up dahmer-nuts-service-space bundy-nuts-service-space >/dev/null &
+VENDOR_FIRST_ID=$(grep -oP "identity: \K(.*)" ./config/bundy/nuts.yaml)
```

Comment:
grep -P is only available on linux, not on mac

Reply:
:-(